libexec/bootstrap-scripts/main-deffile.sh

```diff
@@ -40,6 +40,24 @@ if [ -z "${SINGULARITY_ROOTFS:-}" ]; then
     exit 1
 fi
 
+## For deffile bootstraps, we can implement a "minsize" guard.  This triggers a
+## size check against the image we're trying to bootstrap into, so we don't
+## waste time building into an image file that will be too small.
+#this environment variable might not be defined, so we can make it null.
+MINSIZE=${MINSIZE:-}
+if [ -n "$MINSIZE" ]; then
+  #report that it's been included in the def file.
+  echo "Minimum image size has been set: $MINSIZE M"
```

Comment:
+1 agreed. Good catch. Maybe the "Minimum image size has been set" line should be 1, but the second line with the detected size line should be 2?